fusor-ember-cli/app/controllers/openstack.js

```diff
@@ -1,8 +1,10 @@
 import Ember from 'ember';
 
 export default Ember.Controller.extend({
-  needs: ['undercloud-deploy'],
-  stepNumberOpenstack: Ember.computed.alias("controllers.undercloud-deploy.stepNumberOpenstack"),
+  needs: ['deployment', 'undercloud-deploy'],
+
+  stepNumberOpenstack: Ember.computed.alias("controllers.deployment.stepNumberOpenstack"),
   disableTabRegisterNodes: Ember.computed.alias("controllers.undercloud-deploy.disableTabRegisterNodes"),
-  disableTabAssignNodes: Ember.computed.alias("controllers.undercloud-deploy.disableTabAssignNodes")
+  disableTabAssignNodes: Ember.computed.alias("controllers.undercloud-deploy.disableTabAssignNodes"),
+  validOpenStack: true //TODO
```

Comment:
@jeff-phillips-18, @knowncitizen, what is the criteria for openstack to be valid so the user can continue all to the CFME or Subscriptions. Is this (profiles > 1)? I wasn't sure. We can fix this is a separate PR.